src/openms/source/FORMAT/DATAACCESS/CsiFingerIdMzTabWriter.cpp

```diff
@@ -56,6 +56,44 @@ void CsiFingerIdMzTabWriter::read(const std::vector<String> & sirius_output_path
 
   for (std::vector<String>::const_iterator it = sirius_output_paths.begin(); it != sirius_output_paths.end(); ++it)
   {
+   
+    // extract mz, rt and nativeID of the corresponding precursor spectrum in the spectrum.ms file
+    String ext_nid;
+    double ext_mz = 0.0;
+    double ext_rt = 0.0;
+    const String sirius_spectrum_ms = *it + "/spectrum.ms";
+    ifstream spectrum_ms_file(sirius_spectrum_ms);
+    if (spectrum_ms_file)
+    {
+      const String nid_prefix = "##nid ";
+      const String rt_prefix = "#rt ";
+      const String pmass_prefix = ">parentmass ";
+      String line;
+      while (getline(spectrum_ms_file, line))
+      {
```

Comment:
perhaps trim the line here to avoid errors due to white space

Reply:
I guess it should not be a problem using the hasPrefix() method.